scripts/check-external-layers-providers.js

```diff
@@ -6,16 +6,22 @@ import writeYamlFile from 'write-yaml-file'
 import axiosRetry from 'axios-retry'
 import yargs from 'yargs'
 import { hideBin } from 'yargs/helpers'
-import ogcParser from 'ogc-parser'
 import { exit } from 'process'
-
 import {
     transformUrl,
     isWmsGetCap,
     isWmtsGetCap,
     isKml,
     isGpx,
 } from '@/modules/infobox/utils/external-provider'
+import { WMSCapabilities } from "ol/format";
+import WMTSCapabilities from "ol/format/WMTSCapabilities";
+import { JSDOM } from 'jsdom'
+
+// faking browser support, so that OpenLayers has what it requires to parse XMLs
+const dom = new JSDOM()
+global.DOMParser = dom.window.DOMParser
+global.Node = dom.window.Node
```

Comment:
Hey great I tried to fake the jsdom but couldn't, the example that I used was a bit different from this one and did not worked, that's why I used the ogc-parser lib.

Reply:
maybe a one liner that pushes everything from `dom.window` to `global` is achievable here, would ease things if you require more than theses two API entries